config/permission.php

```diff
@@ -14,6 +14,7 @@
          */
 
         'permission' => Spatie\Permission\Models\Permission::class,
+        'permissions_connection' => env('DB_CONNECTION', 'mysql'),
```

Comment:
Wouldn't it make more sense to use a custom model and use the `connection` instance variable to set the connection? Than we wouldn't need all this code.